packages/web/src/pages/sign-on-page/SignOnPage.tsx

```diff
@@ -0,0 +1,162 @@
+        <SignUpPage />
+      </Route>
+    </Switch>
+  )
+
+  if (isMobile) {
+    return (
+      <Flex direction='column' w='100%'>
+        <Flex
+          h={isExpanded ? '100%' : '520px'}
+          borderBottomLeftRadius={!isExpanded ? '2xl' : undefined}
+          borderBottomRightRadius={!isExpanded ? '2xl' : undefined}
+          css={{
+            position: 'absolute',
+            top: 0,
+            left: 0,
+            width: '100%',
+            backgroundColor: 'white',
+            zIndex: 1,
+            transition: '0.5s ease-in-out 0.5s',
+            transform: isLoaded ? 'translateY(0px)' : 'translateY(-100%)'
+          }}
+        >
+          <Link
+            to={TRENDING_PAGE}
+            css={{ position: 'absolute', left: spacing.l, top: spacing.l }}
+          >
+            <IconCloseAlt color='default' />
+          </Link>
+          {routes}
+        </Flex>
+        <Flex
+          direction='column'
+          alignItems='center'
+          pv='2xl'
+          css={{
+            flexGrow: 1,
+            backgroundImage: `radial-gradient(77.16% 77.16% at 50% 51.81%, rgba(91, 35, 225, 0.80) 0%, rgba(113, 41, 230, 0.64) 67.96%, rgba(162, 47, 235, 0.50) 100%), url(${djPortrait})`,
+            backgroundColor: 'lightgray',
+            backgroundSize: 'cover',
+            backgroundPosition: 'bottom'
+          }}
+        >
+          <Box h={520} />
+          <Switch>
+            <Route path={SIGN_UP_PAGE}>
+              <AudiusValues css={{ margin: 'auto' }} />
+            </Route>
+            <Route path={SIGN_IN_PAGE}>
+              <Text
+                variant='title'
+                strength='weak'
+                color='staticWhite'
+                css={{
+                  marginTop: 'auto',
+                  opacity: isLoaded ? 1 : 0,
+                  transition: 'opacity 0.5s ease-in-out 1s'
+                }}
+              >
+                {messages.newToAudius}{' '}
+                <TextLink variant='inverted' _isHovered asChild>
+                  <Link to={SIGN_UP_PAGE}>{messages.createAccount}</Link>
+                </TextLink>
+              </Text>
+            </Route>
+          </Switch>
+        </Flex>
+      </Flex>
+    )
+  }
+
+  return (
+    <Flex w='100%' p='unit20'>
+      <Link
+        to={TRENDING_PAGE}
+        css={{
+          zIndex: 1,
+          position: 'absolute',
+          left: spacing['2xl'],
+          top: spacing['2xl']
+        }}
+      >
+        <IconCloseAlt color='staticWhite' />
+      </Link>
+      <BackgroundWaves zIndex={0} />
+      <Paper w='100%'>
+        <Box
+          w={isExpanded ? '100%' : 480}
+          h='100%'
+          css={{
+            position: 'absolute',
+            top: 0,
+            left: 0,
+            background: 'white',
+            zIndex: 1,
+            transition: 'width 0.5s ease-in-out'
+          }}
+        >
+          {routes}
+        </Box>
+        <Box w={480} />
```

Comment:
what's this for?

Reply:
same thing, creating a "shadow" bething the absolutely positioned item so the image to the right of it is positioned correctly

Reply:
btw updated these with margin/padding and removed the boxes :)